### PR TITLE
addd unit test for boolean and real64 get methods

### PR DESCRIPTION
**Pull request by injaleea**
[ X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/<reponame>/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Added unit test to cover Get method for boolean and Real64 data types

### Why should this Pull Request be merged?

TODO: Justify why this contribution should be part of the project.

More coverage

TODO: Detail what testing has been done to ensure this submission meets requirements.
Built locally on dev machine

### CODE REVIEW

**Comment by texasaggie97** [comment]
Can one of the admins verify this patch?

**Comment by Fladolcetta** [comment]
ok to test

**Comment by Fladolcetta** [comment]
add to whitelist

File: src/nifake/tests/test_session.py [hunk 1]
```diff
@@ -285,6 +285,30 @@ def test_cannot_add_properties_to_session(self):
             except AttributeError as e:
                 pass
 
+    def test_get_vi_real64_attribute(self):
+        self.patched_library.niFake_GetAttributeViReal64.side_effect = self.side_effects_helper.niFake_GetAttributeViReal64
+        double = 1.5
```
Comment (texasaggie97) [inline src/nifake/tests/test_session.py]: Please don't use the type as a variable name.

File: src/nifake/tests/test_session.py [hunk 2]
```diff
@@ -285,6 +285,30 @@ def test_cannot_add_properties_to_session(self):
...
+            attr_double = session.read_write_double
+            assert(attr_double == double)
+            from mock import call
+            calls = [call(SESSION_NUM_FOR_TEST, b"", 1000001, ANY)]
+            self.patched_library.niFake_GetAttributeViReal64.assert_has_calls(calls)
+            assert self.patched_library.niFake_GetAttributeViReal64.call_count == 1
+
+    def test_get_vi_bool_attribute(self):
+        self.patched_library.niFake_GetAttributeViBoolean.side_effect = self.side_effects_helper.niFake_GetAttributeViBoolean
+        bool = True
```
Comment (texasaggie97) [inline src/nifake/tests/test_session.py]: Same thing. Please don't use the type as a variable name

**Pull request merged**